### ISSUE TITLE
Gitea not remembering installation config

### ISSUE THREAD

**Issue by Hertex** [dockhippie/gitea#4]
- Gitea version (or commit ref): 1.8.3
- Git version: 2.22.0
- Operating system: Alpine Linux 3.10
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [x] MySQL
  - [ ] MSSQL
  - [ ] SQLite

## Description
I am using Gitea with the image webhippie/gitea:latest with docker-compose. Every time I start the setup it will redirect me to the /install page because it lost its previous configuration settings.  I am binding every volume to local directories but it still isn't saving the config. After looking for the app.ini file I noticed that it is not stored on the local directory.

My guess is that you need to add /data/gitea/conf to the list of volumes inside your Dockerfile (might be wrong because I'm no expert in using docker).

**Comment by tboerger** [comment]
There is no such directory /data/gitea. Where did you get that?

**Comment by tboerger** [comment]
Besides that, the installer should never appear, except you have set the value for `GITEA_SECURITY_INSTALL_LOCK` to false.

**Comment by Hertex** [comment]
I set GITEA_SECURITY_INSTALL_LOCK to false because it didn't use the values for my MySQL database that I set with the corresponding variables so I configured it manually.

The directory /data/gitea is being mentioned in the official gitea documentation about the configuration (see [Configuration Cheat Sheet](https://docs.gitea.io/en-us/config-cheat-sheet/). The directory contains the file /data/gitea/conf/app.ini.

**Comment by Hertex** [comment]
I noticed that you are using an individual path and name for the gitea config file (default from you is /etc/gitea/gitea.ini) so I tried to mount this path but it still didn't work.

Now I just recreated my docker-compose file and now it's working properly with the environment variables that you are offering (e.g. GITEA_DATABASE_HOST). Thanks to this I don't need to set GITEA_SECURITY_INSTALL_LOCK to false because I no longer need to set the database manually. I don't know what I did wrong but I am happy that it now works (so it seems that something on my end was wrong and I just didn't get what it was).
